actors/common/src/main/java/org/sunbird/learner/util/OTPUtil.java

```diff
@@ -0,0 +1,97 @@
+package org.sunbird.learner.util;
+
+import com.warrenstrange.googleauth.GoogleAuthenticator;
+import com.warrenstrange.googleauth.GoogleAuthenticatorKey;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.cassandra.CassandraOperation;
+import org.sunbird.common.ElasticSearchUtil;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.ProjectLogger;
+import org.sunbird.common.models.util.ProjectUtil;
+import org.sunbird.common.models.util.datasecurity.EncryptionService;
+import org.sunbird.common.responsecode.ResponseCode;
+import org.sunbird.dto.SearchDTO;
+import org.sunbird.helper.ServiceFactory;
+import org.sunbird.learner.util.Util.DbInfo;
+
+public final class OTPUtil {
+
+  private static CassandraOperation cassandraOperation = ServiceFactory.getInstance();
+  private static EncryptionService encryptionService =
+      org.sunbird.common.models.util.datasecurity.impl.ServiceFactory.getEncryptionServiceInstance(
+          null);
+  private static DbInfo userDb = Util.dbInfoMap.get(JsonKey.USER_DB);
+
+  private OTPUtil() {}
+
+  @SuppressWarnings("unchecked")
+  public static void checkPhoneUniqueness(String phone) {

```

Comment:
Check phone and email uniqueness against cassandra DB. Add these methods in UserService.